streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -176,12 +214,20 @@ private void runSimpleCopyTest(final int numberOfRestarts,
                 startKafkaStreamsAndWaitForRunningState(streams, MAX_WAIT_TIME_MS);
 
                 final List<KeyValue<Long, Long>> inputData = prepareData(i * 100, i * 100 + 10L, 0L, 1L);
+                System.out.println("mjsax: " + inputData.size());
+
+                final Properties producerConfigs = new Properties();
+                if (inputTopicTransactional) {
+                    producerConfigs.setProperty(ProducerConfig.TRANSACTIONAL_ID_CONFIG, applicationId + "-input-producer");
+                    producerConfigs.setProperty(ProducerConfig.RETRIES_CONFIG, "" + Integer.MAX_VALUE);
```

Comment:
Need to set retries no a not-zero value for transactions...

Reply:
The default value for retries are `Integer.MAX_VALUE` anyways right?

Reply:
Yes, but we set it to zero within `TestUtils.producerConfig()` (not sure why) -- should we remove it there instead?

Reply:
Hmm.. I am not sure either but we can check if removing that breaks any other tests --- with the new `deliver` timeout value we should no longer rely on that config values, --- if we want, we should change the `deliver.timeout` not this one.

I guess it's up to you :) if it is too much we can just keep it as is.

Reply:
I removed the `retries` overwrite and left the delivery timeout default. Locally all integration tests passed. Hence, I hope it's fine that way.